mod1/mod11/src/file-5.xoo

```diff
@@ -0,0 +1,11 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// violate HasTag rule: xoo on line 11
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:27685/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:27685/project/issues?id=sample&branch=feature4&issues=AWCrwR5QbEYFRJFExNhY&open=AWCrwR5QbEYFRJFExNhY)

[ISSUE_KEY]: AWCrwR5QbEYFRJFExNhY

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED